src/components/Project/AragonNetwork.js

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import Section from '../General/Section';
+import Entry from './Entry';
+import styled from 'styled-components';
+import {breakpoint, Badge} from '@aragon/ui';
+const medium = css => breakpoint('medium', css);
+
+const AragonNetwork = () => (
+  <RoadmapSection>
+    <Container>
+      <RoadmapContainer>
+        <div className="roadmap-container">
+          <Entry entrypoint>
+            <Title>Current</Title>
+            <Subtitle>
+              Granular areas of focus with well defined scope and product
+              specifications.
+            </Subtitle>
+          </Entry>
+          <Entry>
+            <Module>
+              <div className="badge-container">
+                <EnhancedBadge>Enhanced voting</EnhancedBadge>
+                <AragonOneBadge>Aragon One</AragonOneBadge>
+              </div>
+              <h1>Delegate voting</h1>
+              <h3>
+                Allow people to delegate their voting power to delegates (i.e.
+                representatives) who can vote on their behalf and continue
+                experimenting with new voting mechanisms.
+              </h3>
+            </Module>
+          </Entry>
+          <Entry>
+            <Module>
+              <div className="badge-container">
+                <Badge>Dispute Resolution</Badge>
+                <AragonOneBadge>Aragon One</AragonOneBadge>
+              </div>
+              <h1>Staking and Lock Managers</h1>
```

Comment:
May I suggest to update all the `h1`s into `h2`s? For a given document there should only be one `h1`